fine_tuning/evaluate_finetuned.py

```
@@ -0,0 +1,263 @@
+
+    def _run_predictions(self, df):
+        predictions = []
+        failures = []
+        failure_reviews = []
+
+        for i, (_, row) in enumerate(df.iterrows(), start=1):
+            print(f"   [{i}/{len(df)}] 예측 중...", end='\r')
+            try:
+                pred = self.categorize_single(row['review_text'])
+                predictions.append(pred)
+            except Exception as e:  # pylint: disable=broad-except
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)